schema/input.ts

```diff
@@ -1,3 +1,5 @@
+import { UiConfig } from "./ui";
+
```

Comment:
enum and options are duplicated
also shall we standardize enum / list / options - all seem to be doing the same thing